lib/ash_rbac/policies.ex

```diff
@@ -6,22 +6,22 @@ defmodule AshRbac.Policies do
   use Spark.Dsl.Transformer
 
   alias Ash.Policy.Check.Builtins
+  alias AshRbac.Actions
+  alias AshRbac.Fields
   alias AshRbac.Info
   alias Spark.Dsl.Transformer
 
   @impl true
   def transform(dsl_state) do
-    {field_settings, action_settings} = transform_options(dsl_state)
-
     bypass = Info.bypass(dsl_state)
     bypass_roles_field = Info.bypass_roles_field(dsl_state)
 
     {:ok,
      case Info.public?(dsl_state) do
        false ->
          dsl_state
-         |> add_field_policies(field_settings)
-         |> add_action_policies(action_settings)
+         |> Fields.transform()
```

Comment:
Instead of `transform`, I could also think of `apply`. But don't know if this is a better term/verb ^^

Reply:
It's called transform because it is also implementing the Spark.Dsl.Transformer behaviour. This way, we could potentially just register it as another transformer in the extension instead of calling it ourselves. 

I haven't done that yet because it felt easier to control which transformer is called first. But we could achieve this with the `before`/`after` callbacks as well I think.

Reply:
Ok, nice, good to know! 👍